holoviews/plotting/renderer.py

```diff
@@ -185,7 +185,10 @@ def get_plot(self_or_cls, obj, renderer=None):
             plot_opts = self_or_cls.plot_options(obj, self_or_cls.size)
             plot = self_or_cls.plotting_class(obj)(obj, renderer=renderer,
                                                    **plot_opts)
-            plot.update(0)
```

Comment:
I suppose handling ``default`` means this type of update by integer index is no longer recommended?

Reply:
A user can do what they like I guess, but generally we've internally been moving away from them.